control-plane/gateways/deployment_dataplane_container.go

```diff
@@ -67,7 +67,7 @@ func consulDataplaneContainer(config GatewayConfig, containerConfig v2beta1.Gate
 		Env: []corev1.EnvVar{
 			{
 				Name:  "TMPDIR",
-				Value: "/consul/connect-inject",
+				Value: "/consul/mesh-inject",
```

Comment:
Thoughts on creating a constant for this? Perhaps alongside this guy

https://github.com/hashicorp/consul-k8s/blob/6c7161130276968f03e7f4d8eaff8261c82a3187/control-plane/connect-inject/constants/constants.go#L11-L13